.happy/terraform/modules/batch/variables.tf

```diff
@@ -0,0 +1,46 @@
+variable stack_resource_prefix {
+  type        = string
+  description = "Prefix for account-level resources"
+}
+
+variable app_name {
+  type        = string
+  description = "Batch app name"
+}
+
+variable image {
+  type        = string
+  description = "Image name"
+}
+
+variable batch_role_arn {
+  type        = string
+  description = "ARN for the role assumed by tasks"
+}
+
+variable cmd {
+  type        = string
+  description = "Command to run"
+  default     = ""
+}
+
+variable custom_stack_name {
+  type        = string
+  description = "Please provide the stack name"
+}
+
+variable remote_dev_prefix {
+  type        = string
+  description = "S3 storage path / db schema prefix"
+  default     = ""
+}
```

Comment:
this description is weird to me, i thought this would be whatever we decided to name our remote env (ex: plogan)? When would we be supplying an s3 path?